src/style.css

```diff
@@ -0,0 +1,114 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  font-family: "Roboto", "Arial", "Helvetica", sans-serif;
+  font-style: normal;
+  display: flex;
+  flex-direction: column;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 0 3px 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  margin-bottom: 4px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__product {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: 50%;
+}
+
+.stars :nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
move it to separate file
&
![Screenshot 2022-08-19 at 13 45 08](https://user-images.githubusercontent.com/99331793/185602321-96854ab7-ac32-46f9-bf21-28546d06809e.png)